src/backbone.syphon.js

```diff
@@ -149,7 +153,7 @@ var getForm = function(viewOrForm) {
   if (_.isUndefined(viewOrForm.$el)) {
     return $(viewOrForm).find(':input');
   } else {
-    return viewOrForm.$(':input');
+    return viewOrForm.$(':input, [contenteditable]');
```

Comment:
Have you missed off adding `[contenteditable]` to the `_.isUndefined` case above? ☝️